portal/src/graphql/portal/OnboardingConfigAppScreen.tsx

```diff
@@ -543,9 +543,9 @@ const SecondaryAuthenticationModeContent: React.FC<SecondaryAuthenticationModeCo
     const options: IDropdownOption[] = useMemo(
       () => [
         {
-          key: "required",
+          key: "disabled",
           text: renderToString(
-            "Onboarding.secondary-authentication-mode.required"
+            "Onboarding.secondary-authentication-mode.disabled"
```

Comment:
We may need to implement remove `secondary_authenticators` slices when user select `disabled`. Or we will implement it in the next clean up PR?

Reply:
Also in the on-boarding page, should we hide the mfa question when user select disabled? Or we will do it in the next ui update PR?

Reply:
I will do both of those in this PR, but still pending the new design of the identity/primary authenticators/secondary authenticators configuration screen.